### ISSUE TITLE
Scoop doesn't install

### ISSUE THREAD

**Issue by Bloodroxxor** [lukesampson/scoop#1667]
So i have looked at #118 and #885 before opening this issue. After installing scoop if i run scoop help or anything else it says "scoop : The term 'scoop' is not recognized as the name of a cmdlet, function,
script file, or operable program. Check the spelling of the name, or if a path
was included, verify that the path is correct and try again."

I tried running this script that was posted in the other issue 
`iex (new-object net.webclient).downloadstring('https://gist.githubusercontent.com/lukesampson/210853bdace8e381ff1cb27ef37c8be9/raw/585d44db18b76537e532f16b6e8928587954c5dc/scoop-reinstall.ps1')`

It runs trough fine but same problem as before.
I tried looking for the scoop folder in C:\Users\****\appdata\local> and it is not there.

**Comment by r15ch13** [comment]
You should find it in `C:\Users\<name>\scoop` (%USERPROFILE%) and not in `%LOCALAPPDATA%`.

**Comment by Bladerunner008** [comment]
I had the same issue. I followed the tutorial on the Scoop page and it gave me the same error. 
Then I tried to just open the scoop.cmd file through my normal command prompt (that is without using powershell) and it worked. It gave me the same error, but was still working properly afterwards though.

**Comment by nehalbhanushali** [comment]
I had the same issue. 

I did the following:
* I deleted the contents of the scoop directory,
* Enabled powershell for current user : `set-executionpolicy remotesigned -s currentuser`,
* Installed scoop : `iex (new-object net.webclient).downloadstring('https://get.scoop.sh')`

Just like it is mentioned in the Readme for this repository. The important thing is deleting files that were created due to install without enabling powershell for current user before installing again.
